src/views/StudyEntity/index.tsx

```diff
@@ -53,6 +58,28 @@ const StudyEntity = () => {
             title={intl.get('entities.study.data_access')}
           />
         )}
+
+        {study?.dataset?.hits?.edges?.length && (
```

Comment:
Maybe extract to:

const hasDataset = study?.dataset?.hits?.edges?.length > 0

Reply:
yes en plus ça m'a fait penser j'ai pas géré l'ajout dans le menu aussi qui est conditionnel à ça.